Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-Base/azuredeploy.json

```diff
@@ -0,0 +1,189 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+	"metadata": {
+        "comments": "Establish the needed base resources to leverage with all MDTI playbooks.",
+        "title": "MDTI-Base", 
+        "description": "This playbook creates a shared API Connection for all MDTI playbooks to leverage. This eases the configuration process for a user during deployment of the Microsoft Defender Threat Intelligence(MDTI) solution. In time, this base playbook may be extended to set more functionality. Azure AD App Registration credentials(ClientId/ClientSecret/TenantId) with MDTI API Permissions are needed when configuring this playbook. Those can be found on your [Azure Client App](https://learn.microsoft.com/en-us/rest/api/azure/#register-your-client-application-with-azure-ad) page. If you have trouble accessing your account or your credentials contact your account representative (mdtidiscussion[@]microsoft.com).",
+        "prerequisites": ["None"],
+        "lastUpdateTime": "2023-03-09T00:00:00.000Z", 
+        "entities": [], 
+        "tags": [], 
+        "postDeployment": [
+            "After deploying the playbook, you must authorize the connections leveraged.", 
+            "1. Visit the playbook resource.", 
+            "2. Under 'Development Tools' (located on the left), click 'API Connections'.", 
+            "3. Ensure each connection has been authorized.", 
+            "**Note: If you've deployed the [RiskIQ-Base](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/Microsoft%20Defender%20Threat%20Intelligence/Playbooks/MDTI-Base/azuredeploy.json) playbook, you will only need to authorize the Microsoft Sentinel connection.**"],
```

Comment:
Yes, I'll remove it.

Reply:
Hi @rahul0216 

the MDTI API is not live yet, it will go live on March 28th. thats why you getting a 404 error. the playbooks were tested in CxE Ninja build environment using the Canary Graph API versions.